testing/tests/DevExpress.viz.core/themes.tests.js

```diff
@@ -250,7 +250,6 @@ QUnit.test("Patched properties on register theme", function(assert) {
     assert.strictEqual(theme.treeMap.group.border.color, theme.axisColor, "treeMap - group.border.color");
     assert.strictEqual(theme.treeMap.tile.selectionStyle.border.color, theme.primaryTitleColor, "treeMap - tile.selectionStyle.border.color");
     assert.strictEqual(theme.treeMap.group.selectionStyle.border.color, theme.primaryTitleColor, "treeMap - group.selectionStyle.border.color");
-    assert.strictEqual(theme.treeMap.group.label.font.color, theme.secondaryTitleColor, "treeMap - group.label.font.color");
```

Comment:
Can we fix it without removing?

Reply:
@pavelgruba
LGTY?

Reply:
It doesn't make sense anymore. In this test, we check the patched properties. The TreeMap group label color isn't a patched property anymore (see my comment above).

Reply:
Yes, as I understand now there are different variables which values are used for group label. It used to be one single variable, so we 'patched' it in code.

Reply:
Ok